tests/manual/test-margin-drift.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta http-equiv="content-type" content="text/html; charset=UTF-8">
+		<meta name="viewport" content="initial-scale=1">
+	</head>
+		<body>
+			<div style="position: fixed; border: black solid; width: calc(50% - 10px); min-height: calc(100vh - 20px);">
+				<a href="#">First link</a>
+				<br/><a href="#">Second link</a>
+				<br/><a href="#">Third link</a>
+				<br/><a href="#">Fourth link</a>
+				<br/><a href="#">Fifth link</a>
+				<br/><a href="#">Sixth link</a>
+				<br/><a href="#">Seventh link</a>
+				<br/><a href="#">Eighth link</a>
+				<br/><a href="#">Ninth link</a>
+				<br/><a href="#">Tenth link</a>
+				<br/><a href="#">Eleventh link</a>
+				<br/><a href="#">Twelfth link</a>
+				<br/><a href="#">Thirteenth link</a>
+				<br/><a href="#">Fourteenth link</a>
+				<br/><a href="#">Fifteenth link</a>
+				<br/><a href="#">Second link</a>
+				<br/><a href="#">Third link</a>
+				<br/><a href="#">Fourth link</a>
+				<br/><a href="#">Fifth link</a>
+				<br/><a href="#">Sixth link</a>
+				<br/><a href="#">Seventh link</a>
+				<br/><a href="#">Eighth link</a>
+			</div>
+			<br/>
+			<br/>
+			<br/>
+			<br/>
+			<br/>
+			<div style="position:absolute; border: red solid; left: 33%; width: calc(66% - 10px);">
+			<p/>Test page margins. Left fixed; right scrollable.
```

Comment:
I guess indention is not so accurate here inside `<div>`

Reply:
Thanks, and nicely spotted. It should now be fixed.